safaribooks.py

```diff
@@ -1109,6 +1109,16 @@ def create_epub(self):
         if args_parsed.no_cookies:
             arguments.error("invalid option: `--no-cookies` is valid only if you use the `--cred` option")
 
+    if len(args_parsed.bookid) > 0:
+        bookID = args_parsed.bookid.split("/")[-1]          # Only get book ID from URL
```

Comment:
Makes sense. Can we assure a certain format for the ID (allowed characters, length, ...)?

Reply:
I think that the Book ID can be considered of only digits, then checks if an URL is submitted which contains a Book ID.
Thanks mate!

Reply:
@lorenzodifuccia added the regex check. Please check if it is O.K now.

Reply:
Hi Marcel,
I checked the regex: trying with this URL `https://www.google.com/library/view/test-driven-development-with/9781491958698/`, it will match only 149...
Also, people should be able to paste only the number, without the complete URL, and the current code will always check for an URL